rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -399,8 +399,8 @@ RenderSystem::makeRenderWindow(
 
   params["currentGLContext"] = Ogre::String("false");
 
-  // params["externalWindowHandle"] =
-  //   Ogre::StringConverter::toString(static_cast<unsigned long>(window_id));
+  // This line is needed to receive mouse events on Windows
+  params["externalWindowHandle"] = Ogre::StringConverter::toString(window_id);
```

Comment:
I think this needs to be tested on mac, because if I remember correctly it causes it to segfault.

Reply:
You are right, it does...

Reply:
I wrapped it with a check for Windows now.
Do you want a TODO here as well? I didn't add it for now since there are a lot of platform depending things happening in this class.

Reply:
I think this pr will suss that out: https://github.com/ros2/rviz/issues/75 So, I think this is fine as-is.